src/resolvers/Mutation/updateUserProfile.ts

```diff
@@ -59,7 +59,7 @@ export const updateUserProfile: MutationResolvers["updateUserProfile"] = async (
   // Update User
   const updatedUser = await User.findOneAndUpdate(
     {
-      _id: context.userId,
+      _id: args.data?.id || context.userId,
```

Comment:
why would you allow anyone to update any user's profile they want to?

Reply:
I think this api call would be done by the admin only.

Reply:
this restriction isn't put in place, right now it allows any user to call this mutation

Reply:
This needs to be resolved. Only the profile's user, Admins and Super Admins must be able to edit profile settings.